files/bootstrap.sh

```diff
@@ -269,6 +277,10 @@ if [[ "$MACHINE" != "x86_64" && "$MACHINE" != "aarch64" ]]; then
   exit 1
 fi
 
+if [ "$MOUNT_BPF_FS" = "true" ]; then
+  sudo mount-bpf-fs
```

Comment:
What happens if this is already mounted in the customer's user data? Or via custom AMI that still uses the bootstrap script?

Also, where is `mount-bpf-fs` coming from? Don't see it in this PR.

Reply:
Whoops, missed in my last commit. This process will bail if:
1. There's already a `type bpf` mount.
2. There's already a mount at `/sys/fs/bpf`
3. There's already a file at `/etc/systemd/system/sys-fs-bpf.mount`.